setup.py

```diff
@@ -35,8 +35,8 @@
           url=URL,
           version=VERSION,
           download_url=DOWNLOAD_URL,
-          long_description=open('README.rst').read(),
-          long_description_content_type='text/x-rst',
+          long_description=open('README.md').read(),
```

Comment:
what's the reason for this?

Reply:
I reworked the readme and other parts of the documentation entirely, and GitHub markdown is just easier to read & write than reST. That's the only reason really.